rich/syntax.py

```diff
@@ -193,6 +205,24 @@ def get_background_style(self) -> Style:
         return self._background_style
 
 
+class SyntaxPosition(NamedTuple):
+    """A position in a Syntax object. Lines start at 1, columns at 0"""
+
+    line_number: int
+    column_index: int
+
+
+class SyntaxHighlightRange(NamedTuple):
+    """
+    A range to highlight in a Syntax object.
+    The default style, if none is provided, is to highlight the range in bold.
+    """
+
+    start: SyntaxPosition
+    end: SyntaxPosition
+    style: Style = Style(bold=True)
```

Comment:
Not sure about that one - rather than having such an opinionated (and kinda abstract) default Style, maybe the Style should always be specified? :thinking: